econml/cate_estimator.py

```diff
@@ -381,12 +381,12 @@ def effect(self, X=None, *, T0=0, T1=1):
     effect.__doc__ = BaseCateEstimator.effect.__doc__
 
 
-class StatsModelsCateEstimatorMixin:
+class LinearModelFinalCateEstimatorMixin(BaseCateEstimator):
+    """Base class for models where the final stage is a linear model."""
 
     def _get_inference_options(self):
```

Comment:
I think you can just remove this implementation here.

Reply:
no... we talked about this. Both `StatsModelsCateEstimatorMixin` and `DebiasedLassoCateEstimatorMixin` now inherit from this common base class.

Reply:
I mean the implementation of _get_inference_options. It is already implemented in the BaseCateEstimator and you just repeat it here.

Reply:
oh, gotcha. It shall be removed.